src/providers/details/components/ProviderPageHeaderAlerts.tsx

```diff
@@ -0,0 +1,58 @@
+import type { FC } from 'react';
+import InventoryNotReachable from 'src/modules/Providers/views/list/components/InventoryNotReachable';
+import ProviderCriticalCondition from 'src/modules/Providers/views/list/components/ProviderCriticalCondition';
+
+import type { V1beta1Provider } from '@kubev2v/types';
+import { PageSection } from '@patternfly/react-core';
+import { isEmpty } from '@utils/helpers';
+
+import './ProviderPageHeaderAlerts.style.scss';
+
+type ProviderPageHeaderAlertsProps = {
+  provider: V1beta1Provider;
+  inventoryLoading: boolean;
+  inventoryError: Error | null;
+};
+
+const ProviderPageHeaderAlerts: FC<ProviderPageHeaderAlertsProps> = ({
+  inventoryError,
+  inventoryLoading,
+  provider,
+}) => {
+  const criticalCondition = provider?.status?.conditions?.find(
+    (condition) => condition?.category === 'Critical',
+  );
+
+  const isInventoryNotReachable =
+    provider?.status?.phase === 'Ready' &&
+    !inventoryLoading &&
+    inventoryError &&
+    inventoryError.toString() !== 'Error: Invalid provider data';
+
+  const alerts = [];
```

Comment:
it will casue a lot of renders, as it will be recreated each time something change in the component or props.